src/System.Security.Cryptography.Cng/tests/RsaCngTests.cs

```diff
@@ -0,0 +1,283 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class RsaCngTests
+    {
+        [Fact]
+        public static void EncryptDecryptRoundtrip()
```

Comment:
What I'd like to see in this class is any functionality that is unique to RSACng; like the CngKey ctor.